articles/api-auth/token-introspection.md

```diff
@@ -0,0 +1,81 @@
+---
+description: How to use token introspection to get information about a token's authorization context
+toc: true
+---
+
+# Token Introspection
+
+Auth0 supports [OAuth 2.0 Token Introspection](https://tools.ietf.org/html/rfc7662), which allows you to ask Auth0 for details about any access tokens presented to your API. In addition to asking Auth0 about the *state* of the access token (whether it is active/expired/revoked), you can request information about the token's scopes and the context in which it was issued (such as who issued the token and to whom the token was issued).
```

Comment:
In addition to asking Auth0 about the *state* of the access token (whether it is active or not)